README.md

```diff
@@ -55,12 +55,23 @@ To run Cirrus tasks, simply switch to a directory where the `.cirrus.yml` is loc
 cirrus run
 ```
 
-It is also possible to run a task by name:
+By default, working directory will be `rsync`ed into a container while respecting `.gitignore`
```

Comment:
I think we should move the `--dirty` part of this paragraph after the `cirrus run` example below and provide it's own example to simplify reading.